src/woo_publications/api/authorization.py

```diff
@@ -0,0 +1,16 @@
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import exceptions
+from rest_framework.authentication import TokenAuthentication as _TokenAuthentication
+
+
+class TokenAuthentication(_TokenAuthentication):
+    def authenticate_credentials(self, key):
+        from .models import Application
```

Comment:
I moved the import to the top of the module and things continue to work - why is it a local import?